.gitlab-ci.yml

```diff
@@ -32,7 +32,7 @@ rspec:
   services:
     - postgres:9.5.0
   before_script:
-    - cp config/secrets.example.yml config/secrets.yml
```

Comment:
The file is named secrets.example.yml - are you renaming it?

Reply:
oh, yes, you did. Nevermind.

Reply:
yes